ui/src/views/infra/network/EditTrafficLabel.vue

```diff
@@ -143,8 +143,7 @@ export default {
       })
     }
   },
-  inject: ['parentFetchData'],
```

Comment:
@utchoang Removed by accident ?

Reply:
@davidjumani No, delete as it is not used in this component.

Reply:
Line 198 ?

Reply:
Ah. Maybe I am confused with the PR async job. I'll fix it now.